administration/src/bp-modules/user-settings/ApiTokenSettings.tsx

```diff
@@ -0,0 +1,147 @@
+
+  const appToaster = useAppToaster()
+
+  const [tokenMetaData, setTokenMetadata] = useState<Array<ApiTokenMetaData>>([])
+  const [createdToken, setCreatedToken] = useState<string>('')
+  const [expiresIn, setExpiresIn] = useState<number>(1)
+
+  useEffect(() => {
+    const metaDataQueryResult = getQueryResult(metaDataQuery)
+    if (metaDataQueryResult.successful) {
+      const { tokenMetaData } = metaDataQueryResult.data
+      setTokenMetadata(tokenMetaData)
+    }
+  }, [metaDataQuery])
+
+  const [createToken] = useCreateApiTokenMutation({
+    onCompleted: result => {
+      appToaster?.show({ intent: 'success', message: 'Token wurde erfolgreich erzeugt.' })
+      setCreatedToken(result.createApiTokenPayload)
+      metaDataQuery.refetch()
+    },
+    onError: error => {
+      const { title } = getMessageFromApolloError(error)
+      appToaster?.show({
+        intent: 'danger',
+        message: title,
+      })
+    },
+  })
+
+  const [deleteToken] = useDeleteApiTokenMutation({
+    onCompleted: () => {
+      appToaster?.show({ intent: 'success', message: 'Token wurde erfolgreich gelöscht.' })
+      metaDataQuery.refetch()
+    },
+    onError: error => {
+      const { title } = getMessageFromApolloError(error)
+      appToaster?.show({
+        intent: 'danger',
+        message: title,
+      })
+    },
+  })
+
+  return (
+    <SettingsCard>
+      <H2>Api Token</H2>
+      <Container>
+        <H4>Neues Token erstellen</H4>
+        <p>
+          Ein neu erzeugtes Token wir nur einmalig angezeigt und kann danach nicht wieder abgerufen werden. Bitte
+          speichern sie dieses Token an einem sicheren Ort.
+        </p>
+        <Row>
+          <label htmlFor='expiresIn'>Gültigkeitsdauer:</label>
+          <HTMLSelect
+            name='expiresIn'
+            id='expiresIn'
+            value={expiresIn}
+            onChange={e => setExpiresIn(parseInt(e.target.value))}>
+            <option value='1'>1 Monat</option>
+            <option value='3'>3 Monate</option>
+            <option value='12'>1 Jahr</option>
+            <option value='36'>3 Jahre</option>
+          </HTMLSelect>
+          <Button intent='primary' onClick={() => createToken({ variables: { expiresIn } })}>
+            Erstellen
+          </Button>
+        </Row>
+        {createdToken && (
+          <>
+            <p>Neues Token:</p>
+            <NewTokenText> {createdToken}</NewTokenText>
+          </>
+        )}
+      </Container>
+
+      {tokenMetaData.length > 0 && (
+        <HTMLTable>
+          <thead>
+            <tr>
+              <th>E-Mail des Erstellers</th>
+              <th>Ablaufdatum</th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {tokenMetaData.map(item => (
+              <tr key={item.id}>
+                <td>{item.creatorEmail}</td>
+                <td>{formatDate(item.expirationDate)}</td>
+                <td>
```

Comment:
nit-picky:
maybe we can add vertical alignment in the row to center because currently the delete button and the text are a bit at different levels
![image](https://github.com/user-attachments/assets/46a3d44e-c41a-4906-ae0f-1c3ad80d7ab7)